app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_counter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  after_save :update_post_counter
+
+  private
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] The `five_most_recent_comments` method shouldn't be a private method because it will be called outside this class. I would advise that you make it a public method. The only private method in this class should be the `update_post_counter` method because you don't want that method to be called outside this class but done automatically. That is the reason why you are using the `after_save` method callback to call the method. Kindly implement this change. 👍 😎

Reply:
Fixed.